### DISCUSSION TITLE
Html table bugs

### DISCUSSION THREAD

**Discussion by sbonaime** [requarks/wiki#3802] [Error / Bug Report] [answered]
Hi,
When I paste this code for a simple html table on an empty markdown page

````
<style type="text/css">
.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  overflow:hidden;padding:10px 5px;word-break:normal;}
.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
.tg .tg-wa1i{font-weight:bold;text-align:center;vertical-align:middle}
.tg .tg-11fz{background-color:#38FFF8;text-align:center;vertical-align:middle}
.tg .tg-gskn{background-color:#fe0000;text-align:center;vertical-align:middle}
</style>
<table class="tg">
<thead>
  <tr>
    <th class="tg-wa1i">A</th>
    <th class="tg-11fz">B</th>
    <th class="tg-gskn">C</th>
  </tr>
</thead>
</table>
````

The preview does not show colors of the array
![Capture d’écran 2021-04-06 à 06-04-21 13 47 35](https://user-images.githubusercontent.com/750709/113706355-e1cac300-96de-11eb-8ecd-62d08f773539.png)

If I create a new title after the table, nothing changes.
![Capture d’écran 2021-04-06 à 06-04-21 13 47 49](https://user-images.githubusercontent.com/750709/113706439-f8711a00-96de-11eb-8505-464c13cd8fae.png)

But when I paste the same html table, then the two table get some colors in the preview !
![Capture d’écran 2021-04-06 à 06-04-21 13 47 57](https://user-images.githubusercontent.com/750709/113706494-09ba2680-96df-11eb-860f-e87d8e94689e.png)

Sadly, in any case, colors do not show up in the final rendering
![Capture d’écran 2021-04-06 à 06-04-21 13 48 14](https://user-images.githubusercontent.com/750709/113706567-1fc7e700-96df-11eb-91d9-c242f4307b69.png)

Node.js  12.21.0
PostgreSQL 9.6.20
Version actuelle 2.5.201
Allow HTML is enabled

**Comment by NGPixel** [comment, marked as answer]
CSS code needs to be inserted in the Page Properties dialog, not in the editor.

**Reply by yourjelly** [reply]
> CSS code needs to be inserted in the Page Properties dialog, not in the editor.

CSS inserted in the Page Properties dialog does not get recorded by page history and is wiped when restoring page history.
